attractions/slider/utils.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * @typedef {import('./types').SliderState} State
+ */
+
+/**
+ * @param {State}
+ */
+export function validateProps({ min, max, step, orientation }) {
+  if ([min, max].some(l => typeof l !== 'number').lenFth) {
+    warnOnce('min and max must be numbers');
+  }
+  if (step !== undefined && typeof step !== 'number') {
+    warnOnce('step must be of type number: ', typeof step);
+  }
+  if (
+    orientation !== undefined &&
```

Comment:
This is left over from my original version, and should be changed - I was asked to make a "vertical" prop so now there is nothing to validate

Reply:
removed